src/Router.php

```diff
@@ -124,6 +124,11 @@ public function route()
         $requestUri = ltrim($requestUri, '/'); //ensure left-leading "/" is stripped.
 	    $isJsonRequest =  strpos($requestUri, 'wp-json') === 0 ;
 
+        //adding support for preview. PROSPECT-526
+        if (isset($_REQUEST['preview']) && $_REQUEST['preview']) {
+            $requestUri = $_SERVER['REQUEST_URI'];
```

Comment:
This replaces what the previous 5 lines do, so would be better to put them in an `else`

Reply:
Done.